hack/update-codegen.sh

```diff
@@ -35,3 +35,5 @@ HEADER="# This file is automatically generated. DO NOT EDIT"
 # Add a new CRD? Duplicate these lines
 echo "${HEADER}" > manifests/0000_70_cluster-network-operator_01_pki_crd.yaml
 oc annotate --local -o yaml "${RELEASE_PROFILE}" -f _output/crds/network.operator.openshift.io_operatorpkis.yaml >> manifests/0000_70_cluster-network-operator_01_pki_crd.yaml
+echo "${HEADER}" > manifests/0000_70_cluster-network-operator_05_egr_crd.yaml
+oc annotate --local -o yaml "${RELEASE_PROFILE}" -f _output/crds/network.operator.openshift.io_egressrouters.yaml >> manifests/0000_70_cluster-network-operator_05_egr_crd.yaml
```

Comment:
the final file should be called `0000_70_cluster-network-operator_01_...` - we want to create the CRD before the operator deployment.

Reply:
oh heh thanks for teaching me what the numbering is for XD. I thought its the order of adding new crds or something (feels stupid now that I think about it).